Database/Submit_Full_Regional_Model_SOLA.bat

```diff
@@ -1,538 +1,549 @@
+if exist report.txt (del report.txt /Q)
+CD ..
+
+@ECHO   ***  Cleaning up databank.  ***
+if exist cleanup.rpt (del cleanup.rpt)
+call python useful_macros\cleanup_for_rerun.py %file1% %val%>> cleanup.rpt
+if exist reports (del reports)
+
+REM RUN FREESKIM TO CREATE TIME, DISTANCE AND TOLL MATRICES
+@ECHO.
+@ECHO   ***  Skimming highway network.  ***
+call python prep_macros\free.skim.mac.py %file1% %val%
+if %ERRORLEVEL% neq 0 (goto issue)
+@ECHO.
+
+REM IF PRELOAD=1, REPLACE UNCONGESTED TIME AND DISTANCE MATRICES
+if %preload% EQU 1 (@echo   ***  Preloading congested times and distances.  ***)
+if %preload% EQU 1 (call python prep_macros\preload_congested_times_mac.py %file1% %val%)
+
+@ECHO ==================================================================
+REM - LOOP TO RUN MODEL
+set /A counter=0
+:while
+if %counter% GTR 2 (goto loopend)
+@ECHO -- Begin Transit skim Procedures: %date% %time% >> model_run_timestamp.txt
+@ECHO.
+@ECHO BEGINNING TRANSIT SKIM - FULL MODEL ITERATION %counter%
+@ECHO - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+REM AM Peak Skim
+call python macros/skim_transit.py %file1% %val% %counter% AM
+if %ERRORLEVEL% neq 0 (goto issue)
+rem --  call python macros/transit_triple_indexing.py %file1% AM
+rem --  if %ERRORLEVEL% neq 0 (goto issue)
+rem --  call python macros/transit_skim_final_matrices1.py
+rem --  if %ERRORLEVEL% neq 0 (goto issue)
+rem --  call python macros/transit_skim_wrapup.py %file1% AM
+rem --  if %ERRORLEVEL% neq 0 (goto issue)
+REM Midday Skim
+call python macros/skim_transit.py %file1% %val% %counter% MD
+if %ERRORLEVEL% neq 0 (goto issue)
+rem --  call python macros/transit_triple_indexing.py %file1% MD
+rem --  if %ERRORLEVEL% neq 0 (goto issue)
+rem --  call python macros/transit_skim_final_matrices2.py
+rem --  if %ERRORLEVEL% neq 0 (goto issue)
+rem --  call python macros/transit_skim_wrapup.py %file1% MD
+rem --  if %ERRORLEVEL% neq 0 (goto issue)
+@ECHO    -- End of Transit Skim Procedures: %date% %time% >> model_run_timestamp.txt
+
+@ECHO Begin Global Iteration %counter%: %date% %time% >> model_run_timestamp.txt
+@ECHO PREPARING EMMEBANK - FULL MODEL ITERATION %counter%
+@ECHO - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+rem @ECHO on
+call python macros\init_HOVsim_databk_mac.py %val% %counter% %file1%
+if %ERRORLEVEL% neq 0 (goto issue)
+
+rem Activate Python env
+call %~dp0..\Scripts\manage\env\activate_env.cmd
+
+@ECHO -- Begin Mode-Destination Choice Procedures: %date% %time% >> model_run_timestamp.txt
+@ECHO.
+@ECHO RUN CMAP MODE-DESTINATION CHOICE MODEL - FULL MODEL ITERATION %counter%
+@ECHO - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+call cmap_modedest . --njobs %jobs% --max_zone_chunk %zones%
+if %ERRORLEVEL% NEQ 0 (goto issue)
+@ECHO    -- End Mode-Destination Choice Procedures: %date% %time% >> model_run_timestamp.txt
+@ECHO.
+
+call python macros\transit_tod_vot.py

```

Comment:
transit_tod_vot.py needs to be added to the repo.

Reply:
Made additional edits to Submit_Full_Regional_Model_SOLA.bat and committed to repo: removed call to macros\transit_tod_vot.py and uncommented lines under transit skimming section. These items are in development and were not intended for inclusion in c24q2.